src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -83,10 +102,10 @@
       + "initial connection to discover the full cluster membership (which may change "
       + "dynamically), "
       + "this list need not contain the full set of servers (you may want more than one, though, "
-      + "in case a server is down).";
+      + "in case a server is down). While it is required to configure either of this or "
```

Comment:
Is this extra text necessary? bootstrap.servers is always required, right? I think this is covered off by the comment in the zookeeper connect doc string.